packages/client/src/app/containers/link.js

```diff
@@ -76,6 +77,17 @@ function getHref(props, context) {
       }
     );
 
+  if (
```

Comment:
This is a bit dirty and i would prefer to do this differently. For now i did not find a better way for just changing the paths that need the suffix